software/23.Elixir/README.md

```diff
@@ -19,4 +19,143 @@ Let's help him build the chat app of his dreams ! :)~
 
 ### Understanding Client-Side Rendering vs Server-Side Rendering
 
-TODO
+
+Server-side rendering (SSR) is **an application's ability to convert HTML files on the server into a fully rendered HTML page for the client**. The web browser submits a request for information from the server, which instantly responds by sending a fully rendered page to the client.
+
+On the other hand, with a client-side rendering solution, you redirect the request to a single HTML file and the server will deliver it without any content (or with a loading screen) until you fetch all the JavaScript and let the browser compile everything before rendering the content.
+
+In short, Server-Side Rendering is faster, and more optimized but you serve **static** content, which means you have no interactions with the user. If your user receives a message, he has to reload the whole page, to see the message.
+
+Thankfully using Elixir's [Live View's](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html), there is a way to have the best of both worlds !
+
+## Step 2 -- Building your app
+
+Once everything is installed, we can build our first application.
+To bootstrap your project, you can type the following command: 
+
+```bash
+mix archive.install hex phx_new # Type 'Y' twice
+
+mix phx.new irc_chat --module IrcChat --no-ecto --no-gettext --no-mailer --verbose # type 'Y' once
+```
```

Comment:
You probably need to explane what is mix and more globally, the utility and the fonctionality of these commands because we don't understand really what we are doing with these commands.